### PR TITLE
(maint) Update CODEOWNERS

### PR DESCRIPTION
**Pull request by mwaggett**

### CODE REVIEW

**Comment by Magisus** [comment]
Not sure if this branch has automatic mergeups or not.

**Comment by mwaggett** [comment]
looks like it does? https://jenkins-master-prod-1.delivery.puppetlabs.net/view/puppet-agent/view/mergeups/job/platform_puppet-agent_puppet-agent-vanagon-mergeup_daily-1.10.x/136/

Tests are failing with a deprecation warning though. 🙃

**Comment by Magisus** [comment]
Looks like we don't re-run the component unit tests in jenkins, so it might work to just merge past it.

File: CODEOWNERS [hunk 1]
```diff
@@ -1,15 +1,16 @@
 # default to platform-core
 * @puppetlabs/platform-core
+* @puppetlabs/puppetserver-maintainers
```
Comment (donoghuc) [inline CODEOWNERS]: Isn't there some annoying behavior where @puppetlabs/puppetserver-maintainers will override platform-core?
does it need to be on the same line?
```
* @puppetlabs/platform-core @puppetlabs/puppetserver-maintainers
```

Comment (mwaggett) [reply CODEOWNERS]: oh maybe

Comment (mwaggett) [reply CODEOWNERS]: I forgot about that

Comment (mwaggett) [reply CODEOWNERS]: will fix

**Pull request merged**